test/PointerCasts/multiple_implcit_casts.ll

```diff
@@ -2,12 +2,14 @@
 ; RUN: FileCheck %s < %t.ll
 
 ; CHECK: @test1
-; CHECK: [[gep:%[a-zA-Z0-9_.]+]] = getelementptr %struct.Line, ptr addrspace(1) %lines, i32 %n, i32 1
-; CHECK: store float 0.000000e+00, ptr addrspace(1) [[gep]]
+; CHECK: [[gep1:%[a-zA-Z0-9_.]+]] = getelementptr %struct.Line, ptr addrspace(1) %lines, i32 %n, i32 1
```

Comment:
This is a little awkward, maybe it would be better to add dce to the passes list.